README.md

```diff
@@ -417,6 +417,14 @@ channel.bind('my-event', function () {
 }, { name: 'Pusher' });
 ```
 
+For client-events on presence channels, bound callbacks will be called with an additional argument, an object containing the `user_id` of the user who triggered the event
+
+```
+presenceChannel.bind('client-message', function (data, metadata) {
```

Comment:
```suggestion
presenceChannel.bind('event-name', function (data, metadata) {
```

Reply:
Client events must be prefixed by client-. Events with any other prefix will be rejected by the Channels server, as will events sent to channels to which the client is not subscribed.

https://pusher.com/docs/client_api_guide/client_events#trigger-events